src/components/Header/components/PeopleSearch/index.js

```diff
@@ -152,69 +167,69 @@ export default class GordonPeopleSearch extends Component {
       return null;
     }
     return (
-      <MenuItem
-        {...itemProps}
-        key={suggestion.UserName}
-        component={Link}
-        to={`/profile/${suggestion.UserName}`}
-        onClick={this.reset}
-        className={
-          suggestionList && suggestionList[suggestionIndex] !== undefined
-            ? suggestion.UserName === suggestionList[suggestionIndex].UserName &&
-              suggestionIndex !== -1
-              ? 'people-search-suggestion-selected '
-              : 'people-search-suggestion'
-            : 'people-search-suggestion'
-        }
-      >
-        <Typography variant="body2">
-          {/* If the query contains a space or a period, only highlight occurrences of the first
-              name part of the query in the first name, and only highlight occurrences of the last
-              name part of the query in the last name. Otherwise, highlight occurrences of the
-              query in the first and last name. */}
-          {this.state.highlightQuery.match(/ |\./)
-            ? [
-                this.getHighlightedText(
-                  suggestion.FirstName + ' ',
-                  this.state.highlightQuery.split(/ |\./)[0],
-                ),
-                this.getHighlightedText(
-                  suggestion.LastName,
-                  this.state.highlightQuery.split(/ |\./)[1],
-                ),
-              ].map(e => <span>{e}</span>)
-            : this.getHighlightedText(
-                suggestion.FirstName + ' ' + suggestion.LastName,
-                this.state.highlightQuery,
-              )}
-        </Typography>
-        <Typography variant="caption" component="p">
-          {/* If the first name matches either part (first or last name) of the query, don't
-              highlight occurrences of the query in the first name part of the username.
-
-              If the username contains a period, add it back in.
-
-              If the username contains a period,
-              If the last name matches either part (first of last name) of the query, don't
-              highlight occurrences of the query in the last name part of the username. */}
-          {!suggestion.FirstName.match(
+       <MenuItem
```

Comment:
It looks like this is now indented 3 spaces, instead of 2 as elsewhere.  Is that intentional?  Why?

Reply:
Unintentional. Will revert change with next pull.